packages/forms/stories/story-types.ts

```diff
@@ -131,6 +170,83 @@ export const FIELDSET_ARGS_TYPES = {
   }
 };
 
+export const FILE_UPLOAD_ARG_TYPES = {
+  isHidden: {
+    name: 'Hidden label'
+  },
+  showHint: {
+    name: 'Hint'
+  },
+  showMessage: {
+    name: 'Message'
+  },
+  validation: {
+    name: 'Validation',
+    control: {
+      type: 'radio',
+      options: ['success', 'warning', 'error']
+    }
+  },
+  isDragging: {
+    table: {
+      disable: true
+    }
+  },
+  type: {
+    control: {
+      type: 'select',
+      options: [
+        'pdf',
+        'zip',
+        'image',
+        'document',
+        'spreadsheet',
+        'presentation',
+        'generic',
+        undefined
+      ]
+    }
+  }
+};
+
+export const FILE_LIST_ARG_TYPES = {
+  focusInset: {
+    name: 'Inset File box-shadow'
+  },
+  includeProgress: {
+    name: 'Include Progress'
+  },
+  type: {
+    control: {
+      type: 'select',
+      options: [
+        'pdf',
+        'zip',
+        'image',
+        'document',
+        'spreadsheet',
+        'presentation',
+        'generic',
+        undefined
+      ]
+    }
+  },
+  remove: {
+    name: 'Include Remove',
+    control: {
+      type: 'radio',
+      options: ['File.Close', 'File.Delete', undefined]
+    }
+  },
+  validation: {
```

Comment:
Is it intentional that `validation` for `warning` doesn't have any visual changes, but only when `Message` is true?

![demo](https://gyazo.com/d2b486a675480e2b214d028bf8ede35c.gif)

Reply:
Yes, this is intentional. https://github.com/zendeskgarden/react-components/pull/1249#issuecomment-972895934